install_config/web_console_customization.adoc

```diff
@@ -1014,3 +1014,38 @@ xref:../install_config/install/advanced_install.adoc#advanced-install-configurin
 # See: https://docs.openshift.com/enterprise/latest/install_config/aggregate_logging.html
 #openshift_master_logging_public_url=https://kibana.example.com
 ----
+
+[[configuring-quota-notification-messages]]
+== Configuring Quota Notification Messages
+
+Whenever a user reaches or exceedes a quota a Quota Notification is put into the Notificaiton Drawer.
```

Comment:
These aren't really proper names, so I wouldn't capitalize "quota notification" and "notification drawer."

Reply:
Would be good to link to the quota and limit ranges topic here.